core/src/main/java/org/hibernate/search/jsr352/internal/steps/lucene/EntityReader.java

```diff
@@ -177,7 +177,7 @@ public void open(Serializable checkpointId) throws Exception {
 
 		final int partitionId = Integer.parseInt( partitionIdStr );
 
-		LOGGER.debugf( "[partitionId=%d] open reader for entity %s ...", partitionId, entityName );
+		LOGGER.debugf( "[partitionId=%d] open reader for entity %s ...", (Integer) partitionId, entityName );
```

Comment:
There's a version of `Log.debugf` that accepts an int, but we have build rules in place to forbid its use. I don't remember why exactly, but I think it's because this particular version of the method has been introduced recently and won't work in older versions of WildFly.
Anyway, it just won't build without this cast.